CouchBaseStorage/Configuration/OrleansDocumentExpiry/OrleansGrainExpiryConstants.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace CouchBaseProviders.Configuration.OrleansDocumentExpiry
+{
+    public static class OrleansGrainExpiryConstants
+    {
+        /// <summary>
+        /// Path to target <see cref="OrleansConfigurationSection"/> configuration XML element.
+        /// </summary>
+        public static readonly string OrleansConfigurationSectionPath = "orleans";
```

Comment:
Same as with `OrleansConfigurationSection`, this has the opportunity to clash with other implementations. Also, as you can name them whatever you want and/or nest configsections into groups, maybe we could pass in the path to the Read method?